components/AppBar/examples/AppBarTypicalUseExample.swift

```diff
@@ -19,21 +19,17 @@ import MaterialComponents
 
 class AppBarTypicalUseSwiftExample: UITableViewController {
 
-  // Step 1: Create and initialize an App Bar.
   let appBar = MDCAppBar()
 
   init() {
     super.init(nibName: nil, bundle: nil)
 
     self.title = "App Bar (Swift)"
 
-    // Step 2: Add the headerViewController as a child.
     self.addChildViewController(appBar.headerViewController)
 
-    let color = UIColor(white: 0.2, alpha:1)
-    appBar.headerViewController.headerView.backgroundColor = color
-    appBar.navigationBar.tintColor = UIColor.white
-    appBar.navigationBar.titleTextAttributes = [NSForegroundColorAttributeName : UIColor.white]
+    let mutator = MDCAppBarTextColorAccessibilityMutator()
+    mutator.mutate(appBar)
```

Comment:
Can we keep one of these examples as is to demonstrate both with and without the mutator?

Reply:
Sure. Any preference?